ui/server/src/main/scala/pl/touk/nussknacker/ui/validation/DeploymentCommentValidator.scala

```diff
@@ -0,0 +1,55 @@
+package pl.touk.nussknacker.ui.validation
+
+import cats.data.Validated
+import cats.data.Validated.{Invalid, Valid}
+import pl.touk.nussknacker.ui.listener.DeploymentComment
+
+object DeploymentCommentValidator {
+
+  def createDeploymentComment(comment: Option[String], settings: Option[DeploymentCommentSettings]): Validated[CommentValidationError, Option[DeploymentComment]] = {
+    comment.filterNot(_.isEmpty) match {
```

Comment:
After merging those methods you can easily do `(comment... , settings) match { ... }` and  you reduce level of nesting and will be easier to read this.

Reply:
changed it, thanks